### DISCUSSION TITLE
Anyone used mzMine 3 for polarity switching raw files?

### DISCUSSION THREAD

**Discussion by Tanlinchem** [mzmine/mzmine#1659] [Q&A]
Hello, I am new to mzMine and I tried it today for analyzing our Thermo orbitrap raw files with polarity switching (positive and negative mode). After smoothing, resolved and deisotoped, the peak shape is still looking weird(see below). Anyone had this issue before? Is there any filter I need to apply for polarity switching data?
![image](https://github.com/mzmine/mzmine3/assets/160798151/72f3486c-feb3-424d-9a9a-117dd4df47b7)

**Comment by JethroHemmann** [comment]
Probably you will need to run ADAP chromatogram builder separately for positive and negative mode. There is an option to filter for polarity in the "Scan filters" in the ADAP settings.
